### PR TITLE
resolved conflict about user ID.

### PR DESCRIPTION
**Pull request by Gra2411**

### CODE REVIEW

File: src/main/java/com/agilethought/atsceapi/validator/history/HistoryValidator.java [hunk 1]
```diff
@@ -89,6 +89,12 @@ private void validateHistoryTypeField(Integer type) {
 	}
 
 	private void validateUserId(String user_id) {
+
+		if (user_id == null){
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/validator/history/HistoryValidator.java]: This won't work due to equals comparison with `==` does not work with `String` objects. 
I remember we had a Validation util method that executes a null exception. Please review it.

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

File: src/main/java/com/agilethought/atsceapi/validator/history/HistoryValidator.java (hunk 1 above)
Comment (Gra2411) [reply src/main/java/com/agilethought/atsceapi/validator/history/HistoryValidator.java]: yes! I think i resolved.

**Review by ivanflores15 — APPROVED** [review]
perfect

**Review by luisgalindoluna — APPROVED** [review]
Looks good

**Review by EmiliaGuillen-dev — APPROVED** [review]

**Pull request merged**